api/tests/mocha/services/xforms/multimedia/form.html

```diff
@@ -1,11 +1,11 @@
-
+<root xmlns:xf="http://www.w3.org/2002/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:enk="http://enketo.org/xforms" xmlns:kb="http://kobotoolbox.org/xforms" xmlns:esri="http://esri.com/xforms" xmlns:oc="http://openclinica.org/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
   <form autocomplete="off" novalidate="novalidate" class="or clearfix pages" dir="ltr" id="multimedia">
 <!--This form was created by transforming a OpenRosa-flavored (X)Form using an XSL stylesheet created by Enketo LLC.--><section class="form-logo"> </section><h3 dir="auto" id="form-title">Multimedia - Demo Form</h3><select id="form-languages" style="display:none;" data-default-lang="en"><option value="en">en</option> </select>
   
   
-    <section class="or-group-data or-appearance-field-list " name="g"><label class="question non-select "><video controls="controls" lang="en" class="active" data-media-src="video.mp4" data-itext-id="somevideo">Your browser does not support HTML5 video.</video><input type="text" name="g/q2" data-type-xml="string"/></label><label class="question non-select "><span lang="en" class="option-label active">
```

Comment:
Is this data-media change expected here?

Reply:
Yes. Previously the test wasn't testing enough. Now I've split out the given and expected so we can show that the javarosa tag replacement regex is working as expected.

Reply:
Okay. Thanks for confirming!  The new expected behavior is to output `src="jr://video.mp4"` and no longer output `data-media-src="video.mp4"`.